gateway-service/src/main/java/org/zowe/apiml/gateway/controllers/CacheServiceController.java

```diff
@@ -29,6 +30,7 @@
 @AllArgsConstructor
 @RestController
 @RequestMapping(CacheServiceController.CONTROLLER_PATH)
+@ConditionalOnProperty(name = "apiml.caching.enabled", havingValue = "true")
```

Comment:
It would be this class that needs the conditional: https://github.com/zowe/api-layer/blob/v2.x.x/gateway-service/src/main/java/org/zowe/apiml/gateway/config/CacheConfig.java
Plus, we need to make sure the app will run with it disabled, basically making sure those beans defined in `CacheConfig` have working defaults.

Reply:
fixed